src/main/java/thePackmaster/cards/clawpack/SearingClaw.java

```diff
@@ -0,0 +1,56 @@
+package thePackmaster.cards.clawpack;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.actions.watcher.PressEndTurnButtonAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.combat.ClawEffect;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.CLAW;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class SearingClaw extends AbstractClawCard {
+    public final static String ID = makeID("SearingClaw");
+
+    public SearingClaw() {
+        super(ID, 2, CardType.ATTACK, CardRarity.UNCOMMON, CardTarget.ENEMY);
+        baseDamage = 8;
+        baseMagicNumber = magicNumber = 2;
+        tags.add(CLAW);
+
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+
+        this.addToBot(new VFXAction(new ClawEffect(m.hb.cX, m.hb.cY, Color.ORANGE, Color.WHITE), 0.1F));
```

Comment:
This should totally do one claw effect for each time this card has been upgraded 😃.